roottest/root/multicore/exectsinclude.C

```diff
@@ -22,7 +22,9 @@ void exectsinclude(){
                    "TBrowser.h",
                    "TMatrixD.h",
                    "TF1.h",
+#if __has_include("TGeoSphere.h")
```

Comment:
```suggestion
#ifdef R__HAS_GEOM
```

Reply:
Why? The rest of the file use `__has_include`.

Reply:
Just a suggestion, maybe not worth here. In theory, it makes sense if you have 10 headers of the same library to only put one ifdef rather than 10 if has include. (No idea if it's the case here)

Reply:
It would make sense if we had multiple geometry headers, but since here we only have 1 let's stay consistent with the current code.